content/go/concepts/math-functions/terms/mod/mod.md

```diff
@@ -0,0 +1,113 @@
+---
+Title: 'Mod()'
+Description: 'Returns the floating-point remainder of dividing x by y.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Mod()`** function returns the floating-point remainder of dividing *x* by *y*. The result has the same sign as *x* and a magnitude less than the magnitude of *y*.
+
+The **`Mod()`** function operates exclusively with floating-point numbers.
+
+If the numbers involved in the operation are of integer type, the `%` modulus operator should be used instead. The `%` operator works with integers, while **`math.Mod()`** operates with floating-point numbers.
+
+## Syntax
+
+```pseudo
+math.Mod(x,y)
+```
+
+The **`Mod()`** function takes two arguments of type 'float64'.
+### Some special cases:
+
+1. If the argument has a *+Inf* value, the return value will be *0*:
```

Comment:
generally in the description it may be useful to highlight a formula or some term and I would advise using italics. In the syntax section, and in general, when referring to something in code or a specific language term/pkg/etc use backticks